node/src/chain_spec.rs

```diff
@@ -176,6 +176,8 @@ pub fn local_testnet_config() -> ChainSpec {
 	)
 }
 
+pub const LOCAL_ASSET_ID: u32 = 999;
```

Comment:
I get how `999` feels arbitrary. This actually comes from [`pallets/asset-registry/src/mock.rs`](https://github.com/paritytech/trappist/blob/b31f5d2f93c2c7ad7f8c39b530a7febf14b863dd/pallets/asset-registry/src/mock.rs#L16), where it is important that `LOCAL_ASSET_ID` is different from `StatemineAssetIdInfo`, so that we can be sure that the pallet is doing what it is supposed to do.

I could either set `LOCAL_ASSET_ID` to `1` and `StatemineAssetIdInfo` to something like `10`, or leave `StatemineAssetIdInfo` as `1` and do `LOCAL_ASSET_ID` as `10`.

I'm going with the later option so that we're matching the original slides demo. Let me know if you feel that's still not ideal.